src/frequenz/sdk/timeseries/logical_meter/_formula_engine.py

```diff
@@ -55,14 +55,15 @@ def __init__(
         """Create a `FormulaEngine` instance.
 
         Args:
+            name: A name for the formula.
```

Comment:
oww here is the description :) Should be moved few commits before

Reply:
Fixed.